bool runSenCtrl
bool runVisCtrl
bool ChPolar
float32 PGain
float32 IGain
float32 DGain
float32 SenBias
